LayoutTests/fast/screen-orientation/hidden-document-check.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <script src="../../resources/js-test-pre.js"></script>
+    <script>
+      description(
+        "This test checks that trying to lock or unlock a hidden document results in a SecurityError."
+      );
+
+      var jsTestIsAsync = true;
+
+      async function startTest() {
+        window.internals?.setPageVisibility(false);
```

Comment:
I did not know `?`. Is it equivalent to if (window.internals) window.internals.set... ?

Reply:
I guess the right thing to do should be to stop the test if `window.internals` is not there. There is no point in proceeding if we can't set the page visibility.

Reply:
Either that or if you want to allow manual testing as well in Safari, make it clear through the web page to the manual tester what to do.

Reply:
I went with forcing an error. These tests really shouldn't be run manually.